client/src/components/Profile/Profile.module.css

```diff
@@ -114,4 +114,12 @@
 
 .inputs::-webkit-input-placeholder {
     color: grey;
+}
+
+.loadstate{
+
+      display: "flex";
+      justifyContent: "center";
```

Comment:
`justify-content` will come instead of `justifyContent`. Same goes for `align-items`. I think by mistake you have written these styles this way. Please fix it.

Reply:
Also, `circular progress` is not in center. Verify this from your side and let me know.